pkg/crc/network/proxy.go

```diff
@@ -44,24 +44,24 @@ func NewProxyDefaults(httpProxy, httpsProxy, noProxy, proxyCAFile string) (*Prox
 	}
 
 	if err := ValidateProxyURL(DefaultProxy.HTTPProxy); err != nil {
-		return nil, err
+		return err
 	}
 
 	if err := ValidateProxyURL(DefaultProxy.HTTPSProxy); err != nil {
-		return nil, err
+		return err
 	}
 
 	if noProxy == "" {
 		noProxy = getEnv("no_proxy")
 	}
 	DefaultProxy.setNoProxyString(noProxy)
 
-	return NewProxyConfig()
+	return nil
 }
 
-// NewProxyConfig creates a proxy configuration with the specified parameters. If an empty string is passed
+// GetProxyConfig creates a proxy configuration with the specified parameters. If an empty string is passed
```

Comment:
This comment should be change now, it is now not creating the proxy config but updating the proxyConfig struct which is set by `SetproxyDefaults` and also the env check changes are not part of it now.

Reply:
done :+1: